### PR TITLE
docs: clarify matters related to row existence and the rules for condition evaluation

### PR DESCRIPTION
**Pull request by kostja**

### CODE REVIEW

File: docs/using-scylla/lwt.rst [hunk 1]
```diff
@@ -34,6 +34,92 @@ If any part evaluates to false the transaction does not complete.
...
+            s INT STATIC,
+            PRIMARY KEY(p, c));
+
+It has a partition key ``p``, a clustering key ``c``, a regular
+cell ``r`` and a static cell ``s``.
+
+To materialize a regular row, it's sufficient to assign any of
+``c`` or ``r``:
+
+.. code-block:: cql
```
Comment (tzach) [inline docs/using-scylla/lwt.rst:65]:
```suggestion
.. code-block:: cql

```

File: docs/using-scylla/lwt.rst [hunk 2]
```diff
@@ -34,6 +34,92 @@ If any part evaluates to false the transaction does not complete.
...
+		| [applied]   |   p |   c | s    | r    |
+		|-------------+-----+-----+------+------|
+		| False       |   1 |   1 | null | null |
+		+-------------+-----+-----+------+------+
+
+To materialize a static row, one must explicitly assign
+at least one static cell: otherwise the static row is not
+considered present:
+
+.. code-block:: cql
```
Comment (tzach) [inline docs/using-scylla/lwt.rst:83]:
```suggestion
.. code-block:: cql

```

File: docs/using-scylla/lwt.rst [hunk 3]
```diff
@@ -34,6 +34,92 @@ If any part evaluates to false the transaction does not complete.
...
+		| True        |   1 | null | null | null |
+		+-------------+-----+------+------+------+
+
+It is OK to us a comparison with ``NULL`` in a condition.
+But since ``NULL`` value and missing value in Scylla are
+indistinguishable, conditions which compare with ``NULL``
+will return the same result when applied to both
+missing rows or existing rows with ``NULL`` cells:
+
+.. code-block:: cql
```
Comment (tzach) [inline docs/using-scylla/lwt.rst:103]:
```suggestion
.. code-block:: cql

```

File: docs/using-scylla/lwt.rst [hunk 4]
```diff
@@ -34,6 +34,92 @@ If any part evaluates to false the transaction does not complete.
...
+		| [applied]   | s    |
+		|-------------+------|
+		| True        | null |
+		+-------------+------+
+
+If a regular row is missing, but the static row cells are
+assigned, the static cells will be present in the row used to
+evaluate the condition of the "missing" regular row:
+
+.. code-block:: cql
```
Comment (tzach) [inline docs/using-scylla/lwt.rst:115]:
```suggestion
.. code-block:: cql

```

**Review by tzach — COMMENTED** [review]
Adding a newline after `.. code-block` directive
Without it the table is hidden (see the github [View file](https://github.com/scylladb/scylladb/blob/d24e01be8b1811950fc69402db1660c4cb49ced9/docs/using-scylla/lwt.rst) )

**Review by tzach — APPROVED** [review]

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please backport to 5.1

**Pull request closed without merging**